MapboxMobileEvents.podspec

```diff
@@ -0,0 +1,42 @@
+Pod::Spec.new do |s|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.name = 'MapboxMobileEvents'
+  s.version = '0.2.3'
+  s.summary = 'Mapbox Mobile Events'
+
+  s.description  = 'Mapbox Mobile Events'
+
+  s.homepage = 'https://www.mapbox.com/mobile/'
```

Comment:
This page doesn’t really talk about what this library does. Either the [Telemetry](https://www.mapbox.com/telemetry/) page or the [homepage](https://www.mapbox.com/) would be better.